torch/_inductor/lowering.py

```diff
@@ -44,7 +44,7 @@
     validate_ir,
     View,
 )
-from .utils import ceildiv, decode_device, pad_listlike, sympy_product
+from .utils import ceildiv, decode_device, is_dynamic, pad_listlike, sympy_product
```

Comment:
Seems that `is_dynamic` is unused in this module?

Reply:
No it's used below, I was moving this function out to be used in the ConcatKernel as well.